### PR TITLE
Add new helm chart for a Quorum blockchain explorer & FGT changes

### PR DESCRIPTION
**Pull request by lukasosterheider**

### CODE REVIEW

File: charts/blockchain-explorer/templates/explorer-config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-config
```
Comment (tgip-work) [inline charts/blockchain-explorer/templates/explorer-config.yaml:4]: name should be `{{ include "blockchain-explorer.fullname" . }}`

File: charts/blockchain-explorer/templates/explorer-config.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  namespace: explorer
```
Comment (tgip-work) [inline charts/blockchain-explorer/templates/explorer-config.yaml:5]: namespace should be `namespace: {{ default .Release.Namespace }}`

File: charts/blockchain-explorer/templates/explorer-config.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  annotations:
+    "description": "Configuration of Blockchain Explorer"
+data:
+  config.json: |-
+{{ required "quorumconfig must be set" .Values.quorumConfig | indent 4 }}
+
+  .env.production: |-
```
Comment (tgip-work) [inline charts/blockchain-explorer/templates/explorer-config.yaml:12]: why is it named production?

File: charts/blockchain-explorer/templates/explorer-config.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```
Comment (tgip-work) [inline charts/blockchain-explorer/templates/explorer-config.yaml:1]: Better rename file to "configmap.yaml"

File: charts/blockchain-explorer/values.yaml [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
```
Comment (tgip-work) [inline charts/blockchain-explorer/values.yaml:29]: try to provide secure settings by default

File: charts/blockchain-explorer/values.yaml [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+securityContext: {}
```
Comment (tgip-work) [inline charts/blockchain-explorer/values.yaml:31]: try to provide secure settings by default

File: charts/blockchain-explorer/values.yaml [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```
Comment (tgip-work) [inline charts/blockchain-explorer/values.yaml:59]: Try to set appropriate default resource requests and limits

File: charts/blockchain-explorer/values.yaml [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+affinity: {}
+
+quorumConfig: |-
+    {
+      "algorithm": "qbft",
+      "nodes": [
+        {
+          "name": "rpcnode",
+          "client": "goquorum",
+          "rpcUrl": "http://quorum-node-0-rpc.epi-poc-quorum:8545",
```
Comment (tgip-work) [inline charts/blockchain-explorer/values.yaml:91]: replace `quorum-node-0-rpc.epi-poc-quorum` with `<rpc-servicename>.<quorum-namespace>`

**Review by tgip-work — CHANGES_REQUESTED** [review]

File: charts/blockchain-explorer/Chart.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.2
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.0.0"
```
[the remark this answers is not in the document]
Comment (lukasosterheider) [reply charts/blockchain-explorer/Chart.yaml:24]: The Quorum blockchain explorer has no semantic version es of now. Should I keep it empty then?

File: charts/blockchain-explorer/templates/explorer-config.yaml (hunk 3 above)
Comment (lukasosterheider) [reply charts/blockchain-explorer/templates/explorer-config.yaml:12]: The quorum explorer today just differentiates in ".env.local" if run with "npm run dev" and ".env.production" if run with "npm run build & npm run start".

File: charts/blockchain-explorer/values.yaml (hunk 8 above)
Comment (lukasosterheider) [reply charts/blockchain-explorer/values.yaml:91]: quorum-node-0-rpc.epi-poc-quorum is exactly this naming approach, or?

File: charts/blockchain-explorer/templates/deployment.yaml [hunk 10]
```diff
@@ -38,20 +36,20 @@ spec:
...
             subPath: .env.production
             readOnly: true
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
       volumes:
-      - name: explorer-config-volume
+      - name: config-volume
         configMap:
-          name: explorer-config
+          name: explorer-blockchain-explorer
```
Comment (tgip-work) [inline charts/blockchain-explorer/templates/deployment.yaml:52]: Hi @lukasosterheider : The name must be `{{ include "blockchain-explorer.fullname" . }}`

**Review by tgip-work — CHANGES_REQUESTED** [review]

**Review by tgip-work — APPROVED** [review]

**Pull request merged**